epifor/data/batch.py

```diff
@@ -289,15 +291,52 @@ def write_export_data(self, regions: Regions):
         country into the batch directory.
         """
 
-        out_dir = self.generate_export_dir()
+        out_dir = self.get_out_dir()
```

Comment:
The merge broke my earlier updates to plotting, here and elsewhere ... why did you resolve it this way?